lib/tests/streamlit/elements/exception_test.py

```diff
@@ -62,11 +62,21 @@ def test_markdown_flag(self):
         self.assertFalse(proto.message_is_markdown)
 
         proto = ExceptionProto()
-        exception.marshall(proto, StreamlitAPIException("oh no!"))
+        exception.marshall(
+            proto, RuntimeError("oh no!"), is_uncaught_app_exception=True
```

Comment:
What is the reason for setting `is_uncaught_app_exception=True` here? The test does not seem to need it, but I might miss something.

Reply:
Since `is_uncaught_app_exception` should not affect the results of markdown I included this. I believe `RuntimeError` could be thrown from internal code or during script execution (e.g. from a library dependency). Let me know if that doesn't sound right, should I add a comment?

Reply:
Ah ok, I believe in this case I would parameterize the test and this flag to make it clear that the flag does not have an impact and that the markdown-field in the proto is independent of it. Because when looking at the test right now, it looks like the flag has a meaning and is important for the tests to pass.